code/game/say.dm

```diff
@@ -100,6 +100,10 @@ var/list/freqtospan = list(
 		return "beeps rapidly."
 	else if(message_langs & DRONE)
 		return "chitters."
+	else if(message_langs & SWARMER)
+		return "hums."
+	else if(message_langs & RATVAR)
+		return "sounds like grinding cogs."
```

Comment:
I think the point is that if someone hears a clockwork drone say "un un, genafyngr guvf lbh fpehof", and then spends time translating it themselves, they've either been captured, or they're just being mocked over the radio.

If people really want to spend time putting shit in rot13 translators, and risking admin boinking, then sure. But I think it would be cool to have ratvar language a little more.

Also Xhuis, making a rot13 converter is a very easy task.